tasks/section1.yml

```diff
@@ -804,7 +813,6 @@
       - notscored
       - patch
       - rule_1.5.4
-      - notimplemented
```

Comment:
Why did you remove this tag?

Reply:
These are in my todo list, I msut have modified them, reverted to /bint/true, forgot the tag, it's re-added now